packages/wp-story-editor/src/style.css

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Curious, shouldn't this be marked as a renamed file in Git?

Reply:
Yes, it should, and it is showing a moved file in the parent PR but not after it was moved again in a new folder this PR.